lasagne/layers/dnn.py

```diff
@@ -74,6 +74,11 @@ class Pool2DDNNLayer(Layer):
     def __init__(self, incoming, pool_size, stride=None, pad=(0, 0),
                  ignore_border=True, mode='max', **kwargs):
         super(Pool2DDNNLayer, self).__init__(incoming, **kwargs)
+        if len(self.input_shape) != 4:
+                raise ValueError("Tried to create a 2D pooling layer with "
```

Comment:
Sorry I didn't see this earlier, but this should only be indented by one level (4 spaces), not two levels. (I'm surprised this doesn't violate pep8.) Could you please fix this for the three occurrences? Thanks!